### PR TITLE
Created print.md for Python: Built-in Functions

### LINKED ISSUE #453: [Entry] Python Built-in Functions : print()
### Entry Type (select all that apply)

New Term for Existing Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by KabirCampwala**
### Description

Added print.md as a term entry for Python: Built-in Functions.

Closes #453 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/print/print.md:17]:
```suggestion
The `print()` function takes in a string or any other element such as an integer converted to a string using `str()` and displays those values to the standard output device.
```

I get what this is trying to say 👍 however, it's not very clear at first glance. The `print()` function implicitly calls on `str()`. This is called type casting or type conversion and we have on Docs entry on that: [Python casting](https://www.codecademy.com/resources/docs/python/casting)! I suggest linking out to this source.

Let's re-write this sentence to explain that the `print()` function takes in an argument and displays it to the "standard output device (as you've stated)." Then, explain how the `print()` function works.

**Comment by yangc95** [comment]
Hi @KabirCampwala 👋 I'll be conducting the first review. Thanks for taking on this issue and I'll get back to you shortly❗

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Syntax
+
+```py
+print("Hello, World!")
+```
+
+The output would be:
+
+```py
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/print/print.md:27]:
```suggestion
```pseudo
```

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.
+
+## Syntax
+
+```py
+print("Hello, World!")
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/print/print.md:22]:
```suggestion
print(object)
```

I recommend using a placeholder here. Take a look at the info under the "Syntax" heading for [w3schools](https://www.w3schools.com/python/ref_func_print.asp) or some other (built-in function)[https://www.codecademy.com/resources/docs/python/built-in-functions] Dos entries.

We can create another heading for the examples and include your line 22 as an ex. as well:

## Example 1

Short description:

```py
Code here
```

## Example 2

...and so on

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+If one were to print an integer of a floating-point number they can do so by using the str() function:
+
+```py
+print(str(3.14))
+```
+
+The output would be:
+
+```py
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/print/print.md:39]:
```suggestion
```pseudo
```

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Hello, World!
+```
+
+If one were to print an integer of a floating-point number they can do so by using the str() function:
+
+```py
+print(str(3.14))
```
Comment (yangc95) [inline content/python/concepts/built-in-functions/terms/print/print.md:34]: This works fine, though passing just 3.14 into the function displays the implicit call of `str()`. I'd recommend to not type cast the integer to a string but we can still explain what's going on like how you've done on line 31 for the current line 33-35 codeblock.

**Review by yangc95 — COMMENTED** [review]
Review 1 complete ✅ Please tag me once you've looked over the comments/suggestions @KabirCampwala ☺️

**Comment by yangc95** [comment]
🎄 Ready for review 2️⃣

**Review by yangc95 — APPROVED** [review]

**Comment by Dusch4593** [comment]
@KabirCampwala Hi there! I'll be conducting review 2 and tag you when I'm finished. 😃 Thank you again for this new entry! Much appreciated 🙏

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:14]:
```suggestion
  - 'learn-python-3'
  - 'paths/computer-science'
```

Just one `learn-` slug and one `paths/` slug are sufficient.

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'print()'
+Description: 'Outputs the string representation of an object.'
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:3]:
```suggestion
Description: 'Print the string representation of an object.'
```

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `print()` function takes in a string parameter or any other element, such as an integer. The parameter is converted to a string through an implicit call of the built-in `str()` function and the value is outputted.
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:17]:
```suggestion
The `print()` function accepts an object as a parameter, such as a string, a number, or a list. It is then converted to a string through an implicit call of the built-in `str()` function and the value is printed to an output stream.
```

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+print("Hello, World")
+```
+
+The output would be:
+
+```pseudo
+Hello, World!
+```
+
+**Note:** To print an integer of a floating-point number, use the str() function.
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:37]: Can we expand on this with, perhaps, a codebyte/py example?

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+```py
+print("Hello, World")
+```
+
+The output would be:
+
+```pseudo
+Hello, World!
+```
+
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:36]:
```suggestion

```

File: content/python/concepts/built-in-functions/terms/print/print.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+## Syntax
+
+```py
+print(object)
+```
+
+## Example
+
+```py
+print("Hello, World")
```
Comment (Dusch4593) [inline content/python/concepts/built-in-functions/terms/print/print.md:28]:
```suggestion
print("Hello, World!")
# Output: Hello, World!
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@KabirCampwala All finished with my review! Tag us when you've finished looking over my comments/suggestions, please. 😃

File: content/python/concepts/built-in-functions/terms/print/print.md (hunk 9 above)
Comment (yangc95) [reply content/python/concepts/built-in-functions/terms/print/print.md:31]: @KabirCampwala @Dusch4593 I'm actually a bit confused by this note. When we print an integer or floating-point in Python, can't we just do (for example) `print(3)` or `print(3.14)` which would output 3 and 3.14, respectively?

Comment (Dusch4593) [reply content/python/concepts/built-in-functions/terms/print/print.md:31]: True. Plus, we already have an entry for the `str()` function underway. Gonna remove this and include a codebyte example about using the `sep` and `end` parameters.

**Review by Dusch4593 — APPROVED** [review]
@KabirCampwala Things LGTM for merge but before I do I wanted to let you know more about the changes I recently made: 

- In the "Syntax" section, it's good to have at least one sentence that compliments the code snippet.
- The same thing goes for any "Example" sections. They should not only one a code snippet; there needs to be at least 1 or 2 sentences of supporting text.
- I added a "Codebyte Example" section because for Docs entries, we'd ideally want most of them to have one of these special code snippets near the bottom of the entry. Again, I precluded the snippet with a sentence explaining what's going on in the code.

cc: @yangc95

**Comment by Dusch4593** [comment]
Congratulations, @KabirCampwala! Your entry is now live on the Docs site!

https://www.codecademy.com/resources/docs/python/built-in-functions/print

**Comment by Dusch4593** [comment]
Hey @KabirCampwala, I noticed on the live entry that your name isn't appearing on the list of contributors:

Screen Shot 2021-12-29 at 11 44 57 AM

Did you wish to remain anonymous or receive public credit for this contribution?

**Comment by KabirCampwala** [comment]
@Dusch4593 @yangc95 Thank you very much for your feedback and I hope to do better on the other issue assigned to me. Thank you!

**Comment by KabirCampwala** [comment]
> Hey @KabirCampwala, I noticed on the live entry that your name isn't appearing on the list of contributors:
> 
> Screen Shot 2021-12-29 at 11 44 57 AM
> 
> Did you wish to remain anonymous or receive public credit for this contribution?

I wouldn't mind receiving public credit for this. 😃

**Comment by Dusch4593** [comment]
> I wouldn't mind receiving public credit for this. 😃

@KabirCampwala Excellent! 

First, is your GitHub account connected with your Codecademy account? If not, follow these steps:

1. Go to your Codecademy dashboard and Log in.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. In the Privacy section, under "Profile", select "Everyone" or "Codecademy users" so your account is visible on the Docs entry.
6. Click "Save Changes".

Next, if you'd like, follow these steps to change your Codecademy profile image:

1. Click on your profile image in the top-right and then choose "Profile".
2. Then click "Edit Profile".
3. Click "Choose Photo".
4. Select photo and click "Save Changes".

**Comment by KabirCampwala** [comment]
@Dusch4593 Even though I completed all the steps you've mentioned the contribution is still anonymous.

**Comment by Dusch4593** [comment]
@KabirCampwala Hmmmm let me ask around and find out why that is. 😄

**Comment by KabirCampwala** [comment]
@Dusch4593 Alright! Thank You!!! 🤗

**Comment by Dusch4593** [comment]
@KabirCampwala There may be some internal logic issue regarding how contributors are displayed on the entries. My colleagues and I just id'd another recently merged PR with the same issue as you. I just passed all this info along to my contacts in the Engineering team and we'll hopefully get some more insight into what's going on. I don't expect any updates until next week after the holidays are done, though. But I really appreciate your bringing this to our attention as we want you to get the credit you deserve! 😃

**Comment by KabirCampwala** [comment]
@Dusch4593 Okay no problem.

**Comment by Dusch4593** [comment]
Happy New Year @KabirCampwala! We have an update!

In addition to setting your Codecademy profile visibility to "Everyone", you have to enable SAML single sign-on ([SSO](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on)) between your Codecademy and GitHub profiles _through_ the same primary email address. This can be done by following some steps [here in the CONTRIBUTING.md guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#codecademy-username).

Hope this helps but definitely let us know if you have any questions! 😃

**Comment by KabirCampwala** [comment]
Happy New Year @Dusch4593! Yes I did follow the steps you recommended yesterday and I sent a message as well but I think I forgot to hit send 😝, Thank you @Dusch4593 and @yangc95.

**Pull request merged**